db/migrate/20210409130604_add_bypass_email_login_token_column_to_instructeur.rb

```diff
@@ -0,0 +1,5 @@
+class AddBypassEmailLoginTokenColumnToInstructeur < ActiveRecord::Migration[6.1]
+  def change
+    add_column :instructeurs, :bypass_email_login_token, :boolean, default: false, null: false
```

Comment:
```suggestion
    # This may take a while if running on Postgres < 11
    add_column :instructeurs, :bypass_email_login_token, :boolean, default: false, null: false
```